templates/facets-item-list.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a facets item list.
```

Comment:
This isn't the default implementation. I tend to also remove the variables from the overriding version (we can include additional variables we provide in preprocessors, if any).